blog/2023-08-01-tolgee-with-next-13.md

```diff
@@ -0,0 +1,347 @@
+---
+slug: tolgee-with-nextjs-app-router
+title: 'Tolgee with Next.js app router'
```

Comment:
This is exactly what no-one would search for. People are interested into "Translating Next.js 13 app", "The simplest way to localize Next.js app" etc. I don't even get what's this whole article about from the title.

Reply:
I got like 3 people asking me. Does Tolgee supports next.js router? I think the title is quite alright